bin/spiced/src/main.rs

```diff
@@ -59,7 +59,7 @@ fn main() {
 
     if args.repl {
         if let Err(e) = tokio_runtime.block_on(flightrepl::run(args.repl_config)) {
-            tracing::error!("Flight REPL error: {e}");
+            tracing::error!("Spice SQL REPL Error: {e}");
```

Comment:
Doesn't need Spice (already in that context).

```suggestion
            tracing::error!("SQL REPL Error: {e}");
```